Packs/Campaign/ReleaseNotes/3_3_0.md

```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+
+##### Detect & Manage Phishing Campaigns
+- Fixed a critical issue where a "pepetual lock" would occur, causing all phishing incidents to eventually time-out. The perpetual lock scenario would occur when the **AutomaticallyCloseIncidents** playbook input was set to True, because incidents were closing other incidents which were obtaining the lock, thus resulting in the lock never being released.
+- Fixed an issue where phishing incidents were not added to a closed phishing campaign incident. You can now configure the behavior for incidents related to a closed campaign using the **BehaviorForClosedCampaign** playbook input.
+- Changed the behavior of the **AutomaticallyCloseIncidents** playbook input. When set to True, phishing incidents that have detected a phishing campaign will close themselves instead of closing all detected incidents. As a side-effect of this change, the first few incidents, namely the number specified in the "minIncidentsForCampaign" playbook input, will remain open until their investigation is finished or until they're closed from the Phishing Campaign incident layout.
```

Comment:
```suggestion
- Changed the behavior of the *AutomaticallyCloseIncidents* playbook input. When set to True, phishing incidents that have detected a phishing campaign will close themselves instead of closing all detected incidents. As a side-effect of this change, the first few incidents, namely the number specified in the *minIncidentsForCampaign* playbook input, will remain open until their investigation is finished or until they're closed from the Phishing Campaign incident layout.
```